src/__tests__/standalone.test.js

```diff
@@ -90,6 +90,13 @@ it("standalone with non-existent-file should throw error with code 80", () => {
   })
 })
 
+it("standalone with non-existent-file and allow-empty-input should not throw error", () => {
+  expect(() => standalone({
+    files: `${fixturesPath}/non-existent-file.css`,
+    config: { allowEmptyInput: true },
+  })).not.toThrow()
```

Comment:
Let's also assert the expected outcome. What is the `results` and `output` in `standalone().then(({ results, output })`?